guides/introduction/installation.md

```diff
@@ -183,7 +183,7 @@ However, if you're adding `phoenix_live_view` to an umbrella project, the depend
 }
 ```
 
-Now run the next commands from your web app root:
+Now run the next commands from your project root:
```

Comment:
The current term is more precise because if you have an umbrella app, the web app is a folder inside the umbrella. Perhaps "the root of your web app project:" ?

Reply:
yes! Perfect